main.cpp

```diff
@@ -1,10 +1,58 @@
 #include <iostream>
+#include <iomanip>   // format manipulation
 
 using namespace std;
-
+/**This program displays an equilateral triangle based on the user's input.
+ * The user decides on its size and the character that it may consist of.
+ * The triangle increases by three characters each time and is based on the 
+ * T(n)=2n+1 equation.*/
+void inputCheck(int & size)//Still trying to figure our cin.fail() and cin.good()
```

Comment:
The block comment should be in the following format:

`/**`
  `* The purpose of this function is to  do [what is the function doing]`
  `*`
  `* @param  [data type] [variable name] [variable description]`
  `* @return  [description]`
  `*/`